test/integration/bundles.js

```diff
@@ -48,25 +51,8 @@ describe('Bundles - Integrations', () => {
     await scenario.addAdminAccount(adminAccountWithSecret);
     const nodeAddress = apparatus.identityManager.nodeAddress();
 
-    const kycContract = await apparatus.contractManager.kycWhitelistContract();
-    const HERMES_ROLE_INDEX = 2;
-    await kycContract
-      .methods
-      .add(nodeAddress, HERMES_ROLE_INDEX)
-      .send({
-        from: nodeAddress,
-        gas: 2000000
-      });
-
-
-    const rolesContract = await apparatus.contractManager.rolesContract();
-    await rolesContract
-      .methods
-      .onboardAsHermes(url)
-      .send({
-        from: nodeAddress,
-        gas: 2000000
-      });
+    await addToKycWhitelist(Role.HERMES, apparatus.dataModelEngine, new EmptyLogger());
+    await apparatus.rolesRepository.onboardAsHermes(nodeAddress, url);
```

Comment:
IMO: we can move this sequence of calls into the apparatus. As we will be using them a lot for all kinds of integration tests.